src/patterns/components/inputs/angular/docs/phone.hbs

```diff
@@ -0,0 +1,4 @@
+<p class="sprk-u-mbm">
+  When the field includes a valid input, the sprkFormatterPhoneNumber formatter
```

Comment:
i dont think its necessary to say, since its already covered in the codeinfo table